src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/ProductController.php

```diff
@@ -80,6 +80,19 @@ public function createAction(Request $request): Response
 
             return $this->redirectToRoute('admin_product_new');
         }
+        if ($this->get('prestashop.adapter.multistore_feature')->isUsed()) {
+            return $this->render('@PrestaShop/Admin/Sell/Catalog/Product/disabled.html.twig', [
```

Comment:
Now having a method to display disable message makes sense 

Sorry ... :(

Reply:
like a method that returns `$this->render(...` ?

Reply:
alright 👍

Reply:
Yes, 
if (multistore->isUsed) {
renderDispsablepage
}